salt/modules/pip.py

```diff
@@ -127,7 +125,8 @@ def _process_requirements(requirements, cmd, saltenv, user, no_chown):
                 )
                 __salt__['file.chown'](treq, user, None)
                 cleanup_requirements.append(treq)
-            cmd.append('--requirement={0!r}'.format(treq or requirement))
```

Comment:
This change is causing the pip unit test failures.  We need to test whether pip will accept --requirements='requirements.txt' as a valid argument (instead of --requirements=requirements.txt).